libkbfs/kbfs_ops_test.go

```diff
@@ -3610,3 +3610,39 @@ func TestKBFSOpsBasicTeamTLF(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, u1, ei.LastWriterUnverified)
 }
+
+type wrappedReadonlyTestIDType int
+
+const wrappedReadonlyTestID wrappedReadonlyTestIDType = 1
+
+type wrappedReadonlyNode struct {
+	Node
+}
+
+func (wrn wrappedReadonlyNode) Readonly(ctx context.Context) bool {
+	return ctx.Value(wrappedReadonlyTestID) != nil
+}
+
+func (wrn wrappedReadonlyNode) WrapChild(child Node) Node {
+	return wrappedReadonlyNode{wrn.Node.WrapChild(child)}
+}
+
+func TestKBFSOpsReadonlyNodes(t *testing.T) {
+	config, _, ctx, cancel := kbfsOpsInitNoMocks(t, "test_user")
+	defer kbfsTestShutdownNoMocks(t, config, ctx, cancel)
+
+	config.AddRootNodeWrapper(func(root Node) Node {
+		return wrappedReadonlyNode{root}
+	})
+
+	// Not read-only, should work.
```

Comment:
`t.Log` the comments please.

Reply:
These ones aren't log-worthy in my opinion, I respectfully decline to log them.